packages/aws-cdk-lib/core/lib/private/runtime-info.ts

```diff
@@ -32,14 +35,68 @@ export function constructInfoFromConstruct(construct: IConstruct): ConstructInfo
     && jsiiRuntimeInfo !== null
     && typeof jsiiRuntimeInfo.fqn === 'string'
     && typeof jsiiRuntimeInfo.version === 'string') {
-    return { fqn: jsiiRuntimeInfo.fqn, version: jsiiRuntimeInfo.version };
+    return {
+      fqn: jsiiRuntimeInfo.fqn,
+      version: jsiiRuntimeInfo.version,
+      metadata: isResource(construct) ? redactTelemetryData(construct.node.metadata) : undefined,
```

Comment:
I think [the metadata property](https://github.com/aws/constructs/blob/10.x/src/construct.ts#L271-L273) is the one used for adding the resource metadata. I remember from the design, that we are going to add a new property for the construct, am I correct?

Reply:
No, the latest design solution is to use the existing `metadata` field. This `metadata` is not equivalent and will not result in `Metadata:` field in CFN output template.